.buildkite/x-pack/pipeline.xpack.winlogbeat.yml

```diff
@@ -38,16 +41,16 @@ steps:
           - "x-pack/winlogbeat/build/*.json"
         notify:
           - github_commit_status:
-              context: "Xpack/Winlogbeat Win-2019 Unit (MODULE) Tests"
+              context: "x-pack/Winlogbeat Win-2019 Unit (MODULE) Tests"
```

Comment:
:heart: consistency here helps a ton in having the status results in GH in a sorted order, thank you!